docs/UserGuide.md

```diff
@@ -330,12 +424,35 @@ GuestBook data is saved in the hard disk **automatically** after any command tha
+information at once. By following this incremental approach, we hope to make the content more digestible
+for you.
+
+| Field                   | Corresponding prefix | Constraint                                                                                                                                                                                                    |
+|-------------------------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`NAME`**              | `n/`                 | should only contain alphanumeric characters and spaces.                                                                                                                                                       |
+| **`PHONE`**             | `p/`                 | should only contain numbers, and it should be at least 3 digits and at most 15 digits long.                                                                                                                   |
+| **`EMAIL`**             | `e/`                 | should only contain alphanumeric characters and '@' character. The local segment (before "@") should be at most 64 characters long, and the domain segment (after "@") should be at most 255 characters long. |
+| **`ROOM`**              | `r/`                 | should only contain alphanumeric characters. A hyphen can only be added if it is in between 2 alphanumeric characters.                                                                                        |
+| **`DATE_RANGE`**        | `dr/`                | should follow the format dd/MM/yy - dd/MM/yy.                                                                                                                                                                 |
+| **`NUMBER_OF_GUESTS`**  | `ng/`                | should only contain numbers, and it should be at least 1 and at most 4.                                                                                                                                       |
+| **`REQUEST`**           | `rq/`                | is an optional field, and it should be at most 500 characters long. It should not contain any prefixes.                                                                                                       |
+| **`IS_ROOM_CLEAN`**     | `rc/`                | should only contain `yes`, `no`, `y` or `n` (case-insensitive).                                                                                                                                               |
+| **`BILL`**              | `b/`                 | should only contain numbers, and can have up to 2 decimal places.                                                                                                                                             |
```

Comment:
Should we capitalise the start of each constraint?

Reply:
Fixed!